plugins/network/multibandwidth

```diff
@@ -101,11 +104,20 @@ for host in $hosts; do
                 | cut -d "b" -f1)
 
         if (echo "$SPEED" | grep -q "M"); then
-                echo "$SPEED" | awk '{a+=$1} END{print a*1000000}'
+                VALUE=`echo "$SPEED" | sed 's/.$//'`
```

Comment:
I assume that you want to remove all fractional digits. But I guess, that you forgot to escape the dot?
How about this one for simplicity: `VALUE=${SPEED%.*}`?